components/card/card-loading.component.ts

```diff
@@ -11,15 +11,15 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
   exportAs: 'nzCardLoading',
   template: `
     <div class="ant-card-loading-content">
-      <div class="ant-row" style="margin-left: -4px; margin-right: -4px;" *ngFor="let listOfClassName of listOfLoading">
-        <div
-          *ngFor="let className of listOfClassName"
-          [ngClass]="className"
-          style="padding-left: 4px; padding-right: 4px;"
-        >
-          <div class="ant-card-loading-block"></div>
+      @for (listOfClassName of listOfLoading; track $index) {
+        <div class="ant-row" style="margin-left: -4px; margin-right: -4px;">
+          @for (className of listOfClassName; track $index) {
```

Comment:
Would it be better to use `track: className`?

Reply:
True :) will fix this

Reply:
@HyperLife1119 done :)